packages/design-tokens/docs/types/TokenTuple.ts

```diff
@@ -0,0 +1,3 @@
+import { Token } from "./Token";
+
+export type TokenTuple = [undefined, [string, Token]];
```

Comment:
by changing `buildTokensTableRows` we can simplify it here

```suggestion
export type TokenTuple = [string, Token];
```